examples/views/dropdown.py

```diff
@@ -0,0 +1,38 @@
+import discord
+from discord.ext import commands
+
+class Bot(commands.Bot):
+    def __init__(self):
+        super().__init__(command_prefix=commands.when_mentioned_or('$'))
+
+    async def on_ready(self):
+        print(f'Logged in as {self.user} (ID: {self.user.id})')
+        print('------')
+
+#Gives us a view containing a dropdown menu
+class Dropdown(discord.ui.View):
+    def __init__(self):
+        super().__init__()
+
+    #Send a message pinging the user with their selected choice
+    @discord.ui.select(options=[
+        discord.SelectOption(label="Red"), 
+        discord.SelectOption(label="Green"), 
+        discord.SelectOption(label="Blue")
+    ])
+    async def callback(self, select : discord.ui.Select, interaction : discord.Interaction):
+        await interaction.response.send_message(f"{interaction.user.mention}'s favourite color is {select.values[0]}!")
```

Comment:
```suggestion
        await interaction.response.send_message(f'{interaction.user.mention}'s favourite color is {select.values[0]}!')
```

Reply:
I actually reverted this back to using double quotes since the "'s" halfway through the strings closes the string and throws errors.

Reply:
```suggestion
        await interaction.response.send_message(f'Your favourite colour is {select.values[0]}!', ephemeral=True)
```